src/sentry/models/projectkey.py

```diff
@@ -251,12 +256,12 @@ def get_endpoint(self, public=True):
             urlparts = urlparse(endpoint)
             if urlparts.scheme and urlparts.netloc:
                 endpoint = "{}://{}.{}{}".format(
-                    urlparts.scheme,
+                    str(urlparts.scheme),
                     settings.SENTRY_ORG_SUBDOMAIN_TEMPLATE.format(
                         organization_id=self.project.organization_id
                     ),
-                    urlparts.netloc,
-                    urlparts.path,
+                    str(urlparts.netloc),
+                    str(urlparts.path),
```

Comment:
this is not quite right -- the type checker is telling you these are nullable -- they should be asserted to be non-None rather than converting them to `"None"`